packages/slate-react/src/plugin/react-editor.ts

```diff
@@ -174,7 +174,9 @@ export const ReactEditor = {
 
     return (
       targetEl.closest(`[data-slate-editor]`) === editorEl &&
-      (!editable || targetEl.isContentEditable)
+      (!editable ||
+        targetEl.isContentEditable ||
+        !!targetEl.getAttribute('data-slate-zero-width'))
```

Comment:
adding slate-zero-width characters here means that onDOMSelectionChange will allow this to be a selectable region. The absence of this code causes the deselect routine to be called in the event handler. 

See: https://github.com/ianstormtaylor/slate/blob/master/packages/slate-react/src/components/editable.tsx#L371

Before this change, hasEditableTarget returns false for inline voids (because they're not `isContentEditable=true`)